praw/models/reddit/widgets.py

```diff
@@ -1871,6 +1869,8 @@ def update(self, **kwargs):
             if not key.startswith("_")
         }
         del payload["subreddit"]  # not JSON serializable
+        if "mod" in payload:
+            del payload["mod"]
```

Comment:
Why this change?

Reply:
I need to delete the mod object if it's in the payload, because it cannot be serialized.

Reply:
Is it relevant to fixing the uncached properties? This seems like it belongs in a different PR (maybe one of the widgets PRs).

Reply:
It's relevant to fixing the uncached properties because it didn't used to be an error before making it a cachedproperty.

Reply:
Ah, I see. Thanks for the clarification!